makeingtables/src/Components/Comments.js

```diff
@@ -0,0 +1,43 @@
+import React from 'react'
+import DeleteIcon from '@mui/icons-material/Delete';
+import ImportExportIcon from '@mui/icons-material/ImportExport';
+
+export default class Comments extends React.Component {
```

Comment:
use Component directly , by importing as {component} in line no1

Reply:
done sir